asset/css/default-vars.less

```diff
@@ -1,27 +1,27 @@
-@icinga-blue: #00C3ED;
-@low-sat-blue: #404d72;
-@icinga-blue-dark: #0081a6;
+@primary-button-color: #00C3ED;
+@searchbar-bg-color: #404d72;
+@primary-button-hover-color: #0081a6;
 
 @gray: #c4c4c4;
 @gray-light: #5c5c5c;
 @gray-lighter: #4b4b4b;
-@disabled-gray: #9a9a9a;
+@disabled-color: #9a9a9a;
```

Comment:
Do you have a name suggestion for these variables? 🤔

Reply:
### @disabled-color

These are two variables IMHO: `@selected-search-term-bg` and `@logical-operator-bg`

### @gray-lighter

Also two variables: `@control-hover-bg` and `@suggestions-separation-bg`

### @gray-light

`@disabled-control-color`
`@searchbar-scrollbar-bg`
`@disabled-search-term-bg`
`@selected-search-term-color`
`@search-editor-control-color`
`@search-editor-logical-op-bg`
`@search-editor-context-menu-border-color`
`@card-border-color`

### @gray

`@horizontal-key-color`
`@search-term-bg`
`@search-editor-drag-outline-color`